doc/examples/plot_hyperparameter.py

```diff
@@ -141,13 +169,22 @@ def evaluate_hyperparameters(config):
 # hyerparameters used for that experiment.
 hyperparameters_mapping = {}
 
+###########################################################################
+# Compute accuracy for different randomly generated hyperparameters, and 
```

Comment:
Launch asynchronous parallel tasks for evaluating different hyperparameters. `accuracy_id` is an ObjectID that acts as a handle to the remote task. It is used later to fetch the result of the task when the task finishes.